client/hosting/overview/components/quick-actions-card.tsx

```diff
@@ -18,17 +20,31 @@ import { getSelectedSite } from 'calypso/state/ui/selectors';
 
 interface ActionProps {
 	onClick?: () => void;
+	commandName?: string;
 	icon: ReactNode;
 	href?: string;
 	text: string;
 }
-export const Action: FC< ActionProps > = ( { icon, href, text, onClick } ) => {
+export const Action: FC< ActionProps > = ( { icon, href, text, commandName, onClick } ) => {
 	const isDisabled = ! href && ! onClick;
+	const dispatch = useDispatch();
+	const clickAction = () => {
+		if ( commandName ) {
+			dispatch(
+				recordTracksEvent( 'calypso_hosting_command_palette_navigate', {
```

Comment:
@zaerl I'm not sure if we should be tracking the clicks as if the user executed a [navigation command](https://github.com/Automattic/wp-calypso/blob/efefa75a5e850f4a53f3646fc2a0424353350ed5/packages/command-palette/src/utils.ts#L38). This prevents us from being able to distinguish the click interaction from Site Overview vs. executing a command from the command palette.

Can we change the tracking event to `calypso_overview_quick_action_click` reporting a property `action`?